app/views/estimate_flow/housing_costs.html.slim

```diff
@@ -0,0 +1,65 @@
+- content_for :page_title
+  = t(".heading")
+- content_for :back do
+  = back_link(:housing_costs, @check, @back_buttons_invoke_browser_back_behaviour)
+.govuk-grid-column-two-thirds
+  = form_for(@form, url: wizard_path, method: :put) do |form|
+    = form.govuk_error_summary t("generic.error_summary_title")
+
+    ruby:
+      partner_key = if @check.partner
+                      "heading_hint_partner"
+                    else
+                      "heading_hint"
+                    end
+
+    = render "shared/heading",
+            header_text: t("estimate_flow.housing_costs.heading"),
+            post_header_texts: t("estimate_flow.housing_costs.#{partner_key}")
+    h2.govuk-heading-m class="govuk-!-margin-bottom-2"
+      = t("estimate_flow.housing_costs.housing_payments.label")
+    p.govuk-caption-m class="govuk-!-margin-bottom-4"
+      - t("estimate_flow.housing_costs.housing_payments.hint_paragraphs").each
+        p.govuk-caption-m class="govuk-!-margin-bottom-2" = _1
+
+    = render "shared/money_input",
+            form:,
+            field: :housing_payments,
+            width: 10,
+            label_text: t("estimate_flow.housing_costs.value")
+    = form.govuk_collection_radio_buttons :housing_payments_frequency,
+                                          @form.housing_payment_frequencies,
+                                          :first,
+                                          :last,
+                                          legend: -> { tag.p(t("estimate_flow.housing_costs.frequency")) }
+    h2.govuk-heading-m class="govuk-!-margin-bottom-2"
+      = t("estimate_flow.housing_costs.housing_benefits.label")
+    p.govuk-caption-m class="govuk-!-margin-bottom-4"
+      - t("estimate_flow.housing_costs.housing_benefits.hint_paragraphs").each
+        p.govuk-caption-m class="govuk-!-margin-bottom-2" = _1
+    ul.govuk-list.govuk-list--bullet class="govuk-hint govuk-!-margin-top-0 govuk-!-margin-bottom-0"
+      li = t("estimate_flow.housing_costs.housing_benefits.hint_bullet_1")
+    ul.govuk-list.govuk-list--bullet class="govuk-hint govuk-!-margin-top-1 govuk-!-margin-bottom-0"
```

Comment:
This still isn't quite right. We want one list with 3 items, not 3 lists each with 1 item.

I also think there's too much manual tidging of spacing with `govuk-!-margin` classes.  This is a standard page that should use standard typography in a standard way. If we're having to apply a bunch of manual spacing fixes that either means the design is mis-using the design standards or that we're mis-using the standard components. I think the fixes are as follows:

We shouldn't be using `p.govuk-caption-m` on this screen. We should be using `p.govuk-hint`. The only caption is in the header, which is dealt with in the `shared/heading` partial. If you swap those out, you should find you don't need to do manual spacing adjustment of the `p` elements.

Likewise the `li` elements should be `li.govuk-hint`s and the `ul` should _not_ have the govuk-hint class. That should fix the spacing issues of the bullets

My guess is that the `h2` elements don't need the margin-bottom adjustment classes at all. The designs look pretty much like they don't need it.